docs/integrations/ai-engines/timegpt.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Nixtla's TimeGPT Integration with MindsDB
+sidebarTitle: TimeGPT
+---
+
+TimeGPT by Nixtla is a generative pre-trained model specifically designed for predicting time series data. TimeGPT takes time series data as input and produces forecasted outputs. TimeGPT can be effectively employed in various applications, including demand forecasting, anomaly detection, financial prediction, and more.
+
+You can learn more about its features [here](https://docs.nixtla.io/docs/quickstart).
+
+## How to bring TimeGPT Models to MindsDB
+
+Before creating a model, you will need to create an ML engine for TimeGPT using the `CREATE ML_ENGINE` statement:
+
+```sql
+CREATE ML_ENGINE timegpt
+FROM timegpt;
+```
+
+Once the ML engine is created, we use the `CREATE MODEL` statement to create the TimeGPT model in MindsDB.
+
+```sql
+CREATE MODEL model_name
+FROM data_source
+  (SELECT * FROM table_name)
+PREDICT column_to_be_predicted
+ORDER BY date_column
+GROUP BY column_name, column_name, ...
+WINDOW 12 -- model looks back at sets of 12 rows each
```

Comment:
We can re-use this comments `- model looks back at sets of 12 rows each` for the Neuralforecast tutorials

Reply:
it is there in neuralforecast.mdx and statsforecast.mdx (in the first CREATE MODEL statement which is a template)